ipatests/test_xmlrpc/test_automember_plugin.py

```diff
@@ -528,6 +718,156 @@ def test_rebuild_membership_hosts_group(self, automember_hostgroup, user1,
             command()
 
 
+class TestAutomemberAddConditionNegativeBadgroup(XMLRPC_test):
+
+    def test_create_inclusive_with_nonexistent_group(self, automember_group,
+                                                     group1):
+        group1.ensure_missing()
+        command = automember_group.add_condition(key=u'manager', type=u'group',
+                                                 inclusiveregex=[u'mjohn'])
+        with raises_exact(errors.NotFound(
+                reason=u'Automember rule: %s not found' % group1.cn)):
+            command()
```

Comment:
You can also write a similar test, but with group1.ensure_exists() and automember_group.ensure_missing()